app/views/tag/_subscribe_button.html.erb

```diff
@@ -1,21 +1,22 @@
 <!-- This button is used to display tags in a popover and subscribe to multiple tags using a single button -->
 <% buttonStyle ||= 'btn-outline-secondary btn-md' %>
 <% buttonColor ||= 'black' %>
-<a id="popover_button" class="btn <%= buttonStyle %>" style="color:<%= buttonColor %>;">Follow
+<% index ||= 0 %>
+<a id="popover_button-<%= index %>" class="btn <%= buttonStyle %>" style="color:<%= buttonColor %>;">Follow
   <i class="fa fa-plus" style="color:<%= buttonColor %>;"></i>
 </a>
 
-<div id="mypopcontent" class="d-none" onmouseleave="show(this)">
+<div id="mypopcontent-<%= index %>" class="d-none" onmouseleave="show(this)">
   <%= render partial: "tag/tags_popover", locals: {tags: tags} %>
 </div>
 
 <script type="text/javascript">
-  $('#popover_button').popover({
-    content: $('#mypopcontent').clone().removeClass('d-none'),
+  $('#popover_button-<%= index %>').popover({
+    content: $('#mypopcontent-<%= index %>').clone().removeClass('d-none'),
     placement: 'bottom',
     html: true,
   });
   function show(elem){
-  $('#popover_button').popover("hide")
+    $('#popover_button-<%= index %>').popover("hide")
```

Comment:
Thank you for indenting this @vineethraik!

Reply:
Thank you for providing a chance to be a Open Source contributor. 
If you dont mind could you answer, Why did the check's failed  for the first run ?

Reply:
Hi @vineethraik, other files were using this partial but the `index` variable was not passed through those files as we did in `blog.html.erb`. So I added a check at the top of this page that assigns `index` to `0` if it's `undefined`. Please let me know if you need more explanations.

Sorry for not clarifying earlier!

Reply:
Thank you  for the clarification.